test/tests/globalparams/global_param/tests

```diff
@@ -1,12 +1,20 @@
 [Tests]
   design = 'Parser.md'
-  issues = '#437'
+  issues = '#437 #8671'
 
-  [./global_param_test]
+  [global_param_test]
     type = 'Exodiff'
     input = 'global_param_test.i'
     exodiff = 'out.e'
 
     requirement = 'The system shall include an input file syntax that supports defining global options.'
-  [../]
+  []
+  
+  [test_suppress_ignore]
+    type = 'RunException'
+    input = 'suppresscheck.i'
+    expect_err = "The parameter 'Adaptivity/Markers/boundary/use_displaced_mesh' is a private parameter and should not be used in an input file."
+
+    requirement = 'The system shall not apply global parameter options onto suppressed parameters.'
```

Comment:
this is not the test you want for this. 

This is a good test for "The system shall error if a private parameter is set in the input file"

A good test for that is a RunApp test that does not error

Reply:
you can keep this test, it does not already exist